openslides_backend/action/actions/poll/create.py

```diff
@@ -83,6 +84,10 @@ def update_instance(self, instance: dict[str, Any]) -> dict[str, Any]:
         action_data = []
 
         state_change = self.check_state_change(instance)
+        is_motion_poll = (
+            instance.get("content_object_id")
```

Comment:
`content_object_id` is a required field, therefore checking if `instance.get("content_object_id")` should be unnecessary